ampersand-state.js

```diff
@@ -437,16 +454,17 @@ assign(Base.prototype, Events, {
         }, this);
     },
 
-    _getDerivedProperty: function (name, flushCache) {
```

Comment:
I'm guessing cos it's not used. Doesn't _appear_ to be being used by any other modules as per a github search. What do you think about removing it @HenrikJoreteg? Can you remember if it's useful/needed?